examples/v2/chat2.nim

```diff
@@ -225,6 +250,7 @@ proc publish(c: Chat, line: string) =
             debug "could not append rate limit proof to the message", success=success
           else:
             debug "rate limit proof is appended to the message", success=success
+            echo "--rln epoch: ", fromEpoch(message.proof.epoch)
```

Comment:
For dogfooding we can leave it as echo, we might want to move it to log later though